_includes/com-ingest/com-ingest-extra-kafka-avro-field-syntax.md

```diff
@@ -0,0 +1,74 @@
+### Kafka Avro data type syntax
+
+Map Avro field data types and property key-value pairs to determine the FeatureBase record data type
+
+#### BOOL
+
+| Kafka Avro fields | Description |
+|---|---|
+| `{"name": "bool_bool", "type": "boolean"}` | FeatureBase Bool from Avro Boolean
+
+#### DECIMAL
+
+| Kafka Avro fields | Description |
+|---|---|
+| `{"name": "decimal_float", "type": "float", "fieldType": "decimal", "scale": 2}` | FeatureBase Decimal from Avro Float |
+
+#### ID
+
+| Kafka Avro fields | Description |
+|---|---|
+| `{"name": "id_long", "type": "long", "mutex": true, "fieldType": "id"}` | FeatureBase ID from Avro Long |
+| `{"name": "id_int", "type": "int", "mutex": true, "fieldType": "id"}` | FeatureBase ID from Avro int |
+
+#### IDSET
+
+| Kafka Avro fields | Description |
+|---|---|
+| `{"name": "idset_int", "type": "int", "fieldType": "id"}` | FeatureBase IDSET from Avro Int |
+| `{"name": "idset_intarray", "type": {"type": "array", "items": "int"}}` | FeatureBase IDSET from Avro Int Array |
+
+#### IDSETQ
+
+| Kafka Avro schema | Kafka Avro fields | Description |
```

Comment:
This is a stuff-up/mistake because of the RecordTime content found here: https://docs.featurebase.com/docs/community/com-ingest/com-ingest-kafka-avro/#stringsetq

Specifically this content:

```
Note this must be accompanied with a RecordTime field in the avro schema. For example:

{"name": "recordtime_bytes", "type": "bytes", "fieldType": "recordTime", "layout": "2006-01-02 15:04:05", "unit": "s"},
```

I'll recreate the table from original content in https://docs.featurebase.com/docs/community/com-ingest/com-ingest-kafka-avro/#idset

Reply:
Looks good to me!

Reply:
For the win :D